### ISSUE TITLE
Feature Request: disabled item prop, selectedItemStyle and disabledItemStyle props

### ISSUE THREAD

**Issue by kodie** [n4kz/react-native-material-dropdown#79]
It would be nice to be able to do something like this:

```
<Dropdown
  label='Please select an item'
  data={[
    { value: 'item1', label: 'Item 1' },
    { value: 'item2', label: 'Item 2', disabled: true },
    { value: 'item3', label: 'Item 3' }
  ]}
  selectedItemStyle={ { backgroundColor: 'green', color: 'white' } }
  disabledItemStyle={ { backgroundColor: 'red', color: 'grey' } }
/>
```

The closest you can get currently is doing something like this:
```
propsExtractor={ (props) => { return { disabled: props.disabled } } }
```
However you can't style it differently at all.

**Comment by n4kz** [comment]
Thanks for issue! It's possible to get different styles by returning [`Button`](https://github.com/n4kz/react-native-material-buttons) component's props from `propsExtractor`.

**Comment by kodie** [comment]
@n4kz Using `propsExtractor` only allows you to change the background color, you have no control over text color with that. Also, if you pass `disabled: true` to the button, it doesn't allow you to scroll past disabled items, touching them at all causes the dropdown to close.

**Comment by n4kz** [comment]
Okay, I will implement style merging in next release to allow style overrides from `propsExtractor`. Also will take a look on touch handling for disabled items.

**Comment by n4kz** [comment]
Released `0.11.0` with few improvements:

- Touches on disabled items are ignored
- Disabled items have transparent background by default
- `style` received from `propsExtractor` is added to item styles
- Added `disabledItemColor` prop to `Dropdown` component

**Comment by comanzanares** [comment]
Can anyone provide an example code of how to change selected item background color?
